fuse_variables/include/fuse_variables/orientation_3d_stamped.h

```diff
@@ -127,17 +139,21 @@ class Orientation3DLocalParameterization : public fuse_core::LocalParameterizati
     return true;
   }
 
+#if CERES_VERSION_AT_LEAST(2, 1, 0)
+  bool MinusJacobian(
+#else
   bool ComputeMinusJacobian(
+#endif
     const double* x,
     double* jacobian) const override
   {
     double x0 = x[0] * 2;
     double x1 = x[1] * 2;
     double x2 = x[2] * 2;
     double x3 = x[3] * 2;
-    jacobian[0] = -x1; jacobian[1]  =  x0; jacobian[2]  =  x3;  jacobian[3]  = -x2;  // NOLINT
```

Comment:
@nlunscher-cpr Same as above, but here it's for the 3D orientation instead of the 2D one.

The correct jacobian simply has a different sign. That's because the `Minus` operator is computing `y_minus_x = y - x`, and the jacobian was incorrectly computed wrt `y` before, and now is computed wrt `x`, which is the same jacobian but with different sign.